test/Microsoft.ML.AutoML.Tests/DatasetUtil.cs

```diff
@@ -82,5 +84,124 @@ private static string DownloadIfNotExists(string baseGitPath, string dataFile)
 
             throw new Exception($"Failed to download test file {dataFile}.");
         }
+
+        public static string GetFlowersDataset()
+        {
+            const string datasetName = @"flowers";
+            string assetsRelativePath = @"assets";
+            string assetsPath = GetAbsolutePath(assetsRelativePath);
+            string imagesDownloadFolderPath = Path.Combine(assetsPath, "inputs",
+                "images");
+
+            //Download the image set and unzip
+            string finalImagesFolderName = DownloadImageSet(
+                imagesDownloadFolderPath);
+
+            string fullImagesetFolderPath = Path.Combine(
+                imagesDownloadFolderPath, finalImagesFolderName);
+
+            var images = LoadImagesFromDirectory(
+                folder: fullImagesetFolderPath, useFolderNameAsLabel: true);
+
+            using (StreamWriter file = new StreamWriter(datasetName))
+            {
+                file.WriteLine("Label,ImagePath");
+                foreach (var image in images)
+                    file.WriteLine(image.Label + "," + image.ImagePath);
+            }
+
+            return datasetName;
+        }
+
+        public static IEnumerable<ImageData> LoadImagesFromDirectory(string folder,
+            bool useFolderNameAsLabel)
+        {
+            var files = Directory.GetFiles(folder, "*",
+                searchOption: SearchOption.AllDirectories);
+            foreach (var file in files)
+            {
+                if (Path.GetExtension(file) != ".jpg")
```

Comment:
Incase someone sees and replicates this code...

Recommend: .jpg, .jpeg, .png, .gif. 
And case insensitive. 
```suggestion
                var extension = Path.GetExtension(file).ToLower();
                if (extension != ".jpg" &&
                    extension != ".jpeg" &&
                    extension != ".png" &&
                    extension != ".gif"
                )
```